scripts/tests/test-ci-all.sh

```diff
@@ -224,7 +224,7 @@ parsed_test_commands=$(printf "%s\n" "${tests_with_versions[@]}")
 parallel_args=()
 export parallel_jobs='1'
 
-if [ -z "${CI:-}" ]; then
+if [ -z "${CI:-}" ] && [[ -t 1 ]] && [ -z "${FM_TEST_CI_ALL_DISABLE_ETA:-}" ]; then
```

Comment:
Can we remove the CI env var check now that there's the file descriptor 1 check?

Reply:
I don't know. It seems to make sense to disable it if CI is set... kinda... too lazy to to change something that I don't have a strong feeling for...